src/resources/FileUpload.js

```diff
@@ -3,6 +3,7 @@ const AWS = require("aws-sdk");
 const debug = {
   error: require("debug")("formio:error")
 };
+const FormioConfig = require('../../config/default.json');
```

Comment:
Consider renaming to `config` instead of `FormioConfig`

Reply:
Renamed variable to config